sharding/client/utils.go

```diff
@@ -0,0 +1,39 @@
+package client
+
+import (
+	"fmt"
+)
+
+type collationbody []byte
+
+var (
+	collationsizelimit = int64(2 * *20)
+	chunkSize          = int64(32)
+	indicatorSize      = int64(1)
+	chunkDataSize      = chunkSize - indicatorSize
+)
+
+func (c *collationbody) validateBody() bool {
+	return len(c) < 2^20 && len(c) > 0
```

Comment:
Inline magic numbers like this aren't understood by outside viewers. Please store in vars called maxcollationsize, mincollationsize, or another sort of construct.

Reply:
Agreed, will put a variable in place and maybe move all the constants to config.go

Reply:
I left a comment on the design as well, but this would never consider a body valid when the `len(cb) == 2^20` i.e. when the collation body is exactly 1 MiB